packages/cubejs-server-core/src/core/DevServer.ts

```diff
@@ -40,11 +40,13 @@ export class DevServer {
     const cubejsToken = jwt.sign({}, options.apiSecret || 'secret', { expiresIn: '1d' });
 
     if (process.env.NODE_ENV !== 'production') {
-      console.log('🔓 Authentication checks are disabled in developer mode. Please use NODE_ENV=production to enable it.');
+      console.log('Cube.js is running in development mode: https://cube.dev/docs/configuration/overview#development-mode');
```

Comment:
```suggestion
      console.log('Cube.js is running in development mode:\nhttps://cube.dev/docs/configuration/overview#development-mode');
```

Reply:
Let's keep it as is then. Terminal emulators will do the job terminating the lines for us :)